_posts/2025-12-07-openttd-15-0-rc1.md

```diff
@@ -0,0 +1,26 @@
+---
+title: OpenTTD 15.0-rc1
+author: michi_cc
+---
+
+Good things come to those who wait. OpenTTD 15 is starting the final lap with first release candidate!
+
+With a release candidate, the focus shifts from adding new features to finding and fixing bugs to make OpenTTD 15 the best OpenTTD yet.
+The finding part is where you, the players, come in, too. If there's anything you notice, head over to our [issue tracker](https://github.com/OpenTTD/OpenTTD/issues/new/choose) and make a report.
+
+Some notable changes over the previous beta3 release are:
+* Automatic font loading for characters not part of the normal language files (like e.g. player names or chat messages)
+* Terrain generator can make wetlands for rivers unable to reach the sea
+* Signs, waypoint and station names may be moved
+* Draw infinite water when all borders are water
+* Bridges can now be built over locks and docks, too
+* Various tweaks to interface scaling
+* Improvements and fixes to badge support
```

Comment:
Should this be tagged NewGRF?
```suggestion
* [NewGRF] Improvements and fixes to badge support
```

Reply:
Not sure, most are UI fixes, only one actual NewGRF addition.